src/StringParser/Common.purs

```diff
@@ -1,6 +1,6 @@
 -- | This module defines the `Parser` type of string parsers, and its instances.
 
-module Text.Parsing.StringParser where
+module StringParser.Common where
```

Comment:
Maybe this should be `StringParser.Parser` since it specifically deals with the parser type?

Reply:
I named it Common because it contains more than just the Parser type, namely all the things that the CodeUnits parser and the CodePoints parser share but are not in Combinators.

But you have a lot more knowledge about naming conventions in the ecosystem, so should it be Parser instead? You tell me :smile:

Reply:
I may be misunderstanding you; all I see in this file is the parser type and instances, plus `runParser` and `unParser` for running the parser, `fail` for failing a parser, and `printParserError` for unpacking the error message.

> namely all the things that the CodeUnits parser and the CodePoints parser share but are not in Combinators.

I guess my question is — what is being shared here beyond the `Parser` type and helpers for its error messages? If it's nothing, then I think `StringParser.Parser` is a more informative module name. If it's more than that, then `Common` makes more sense.

Reply:
> what is being shared here beyond the Parser type and helpers for its error messages?

That's pretty much it as you can see. So it's decided then :)